src/modules/drawing/components/DrawingStylePopup.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+    <div v-if="feature" class="card drawing-style-popup">
+        <div class="card-header">
+            <span class="float-left">{{ $t('draw_popup_title_feature') }}</span>
+            <button type="button" class="close" aria-label="Close" @click="onClose">
+                <span aria-hidden="true">&times;</span>
+            </button>
+        </div>
+        <div class="card-body text-left">
+            <form>
+                <div v-if="feature.geometry.type === 'Point'" class="form-group">
+                    <label for="text">{{ $t('draw_popup_title_annotation') }}:</label>
+                    <textarea id="text" v-model="text" class="form-control" rows="1"></textarea>
+                </div>
+                <div class="form-group">
+                    <label for="description">{{ $t('modify_description') }}:</label>
+                    <textarea
+                        id="description"
+                        v-model="description"
+                        class="form-control"
+                        rows="2"
+                    ></textarea>
+                </div>
+                <div class="form-group p-0 button-bar">
+                    <geometry-measure :geometry="feature.geometry"></geometry-measure>
+                    <button type="button" class="btn btn-default" @click="onDelete">
+                        <font-awesome-icon :icon="['far', 'trash-alt']" />
+                    </button>
+                </div>
+            </form>
+        </div>
+    </div>
+</template>
+
+<script>
+import GeometryMeasure from './GeometryMeasure.vue'
+
+export default {
```

Comment:
Please add a little JSDoc explaining briefly what this component is doing / its responsibilities